src/runtime/cli/config/configuration-clh.toml.in

```diff
@@ -93,6 +93,17 @@ virtio_fs_cache = "@DEFVIRTIOFSCACHE@"
 # or nvdimm.
 block_device_driver = "virtio-blk"
 
+# This option enables SGX. Hardware-based isolation and memory encryption
+# Comma-separated list of Enclave Page Cache (EPC) sections.
+# A EPC section has two colon-separated properties: `size:prefault`.
+# `size` is the size of the EPC section and must be aligned to 4k.
```

Comment:
- What suffixes can users use? Only "M", or "K" / "G" / "T" ?
- Can a pure numeric be specified and if so is it assumed to be "M"?